src/rules/get-fullcontact-profile.js

```diff
@@ -6,46 +6,50 @@
  *
  * This rule gets the user profile from FullContact using the e-mail (if available).
  *
- * If the information is immediately available (signaled by a `statusCode=200`), it adds a new property `fullContactInfo` to the user_metadata and returns. Any other conditions are ignored.
+ * If the information is immediately available (signaled by a `statusCode=200`), it adds a new property `fullcontact` to the user_metadata and returns. Any other conditions are ignored.
  *
- * See [FullContact docs](http://www.fullcontact.com/developer/docs/) for full details.
+ * See [FullContact docs](https://dashboard.fullcontact.com/api-ref) for full details.
  *
  */
 
 function (user, context, callback) {
-  const FULLCONTACT_KEY = configuration.FULLCONTACT_KEY;
-  const SLACK_HOOK = configuration.SLACK_HOOK_URL;
+  // Get FullContact API key: https://dashboard.fullcontact.com/
+  const FULLCONTACT_KEY = 'YOUR FULLCONTACT API KEY'; 
```

Comment:
Any particular reason why the support for `configuration` variables are removed here for `FULLCONTACT_KEY` and `SLACK_HOOK`?

Reply:
Honestly, I just couldn't find a way to pass these as variables from elsewhere, and it looked more explicit and direct to just have these inline "instructions" for the user of what to put where in the rule to make it work. But this is subjective, of course.

Reply:
There is a global configuration object that is available for the rules and that's where the API keys and webhook URLs can be stored. Please see https://auth0.com/docs/rules/guides/configuration.

Reply:
Oops, missed it completely. My bad. Just pushed an update.

Reply:
Awesome! Many thanks for the PR!